core/src/main/java/org/arquillian/smart/testing/api/SmartTesting.java

```diff
@@ -23,7 +24,7 @@
      * @return An instance of {@link ConfiguredSmartTesting} class
      */
     public static ConfiguredSmartTesting with(TestVerifier testVerifier) {
-        return with(testVerifier, Configuration.load());
+        return with(testVerifier, Configuration.loadPrecalculated(Paths.get("").toFile()));
```

Comment:
As a default behavior, I would load the file from main directory - the original one. `Configuration.load(Paths.get("").toFile())`
The precalculated is Maven(surefire) specific.

Reply:
No this `with` method is not used anywhere. We are using with `(TestVerifier testVerifier, Configuration configuration)`. You can look [here](https://github.com/arquillian/smart-testing/pull/183/files/99430c1dc61dcd34884d69229d0eaf08c30e1210#diff-e92614f0bf12fbca05272c1456ecba67R67). I am not sure about `che` for this. Is it used or not?

Also one more thing, as with default pointing to `rootDir` other submodule won't have this configuration file in it's respective directory, So it won't work

Reply:
It had been used somewhere in ST, but not now obviously. 
BTW, the API is not intended to be used for one parent module expecting that it will invoke all submodules.
Nevermind, leave it as it is.